torch/_inductor/codegen/triton.py

```diff
@@ -756,6 +756,8 @@ def indexing(
                 # indirect indexing
                 cse_var = self.cse.varname_map[var.name]
                 mask_vars.update(cse_var.mask_vars)
+            elif var.name.startswith("s"):
```

Comment:
this is cursed - is this really how we discover symbolic shape vars?

Reply:
Yeah, the symbol prefixes are informative. You can see that the below code verifies that the symbol prefix is in "xyr".